docs/setup.md

```diff
@@ -2,7 +2,7 @@
 
 This guide will help you set up refreshVersions in a Gradle project.
 
-## Ensure Gradle is up to date
+## Update Gradle
```

Comment:
This is outside of the scope of what the commit title specifies.

Why this change?